nss-day-cohort-48/rare-client-boastful-bulldogs
# Categories

Categories and Categories Manager

## Steps to Test:
1. git fetch -all
2. git checkout hm-createcat-c (client); hm-createcat-s (server)
3. click on Categories Manager in navbar
4. see categories listed
5. click on "Create Category" button
6. input new category title
7. should route you back to the categories manager page where you can see the list of categories including the new one.
